### DISCUSSION TITLE
Double Quote vs Single Quote preference

### DISCUSSION THREAD

**Discussion by Glatinis** [community/community#10854] [Copilot Conversations]
All the code suggestions I get for Python use single quotes, while I personally use double quotes. There should be a way for the user to configure that.

**Comment by Yash-Singh1** [comment]
GitHub Copilot adapts and learns to how you write your code, so it should learn after some time.

**Comment by KTibow** [comment]
Remember, GitHub Copilot is an AI, and an expensive one at that. You can't just make two versions of Copilot.

Personally, I'd recommend for you to install a formatter extension, so that the quotes get swapped to your liking (the Python VS Code extension has support for auto-formatting).

**Comment by thistlillo** [comment]
I second this request.

I have been using github copilot for several weeks and it has not yet learnt the quotes I use. I normally use double quotes but suggested expansions are always with single quotes.

**Reply by KTibow** [reply]
Could you send an example of your code? Copilot cannot "learn" the quotes you use over time; it's not adaptive to yourself. However, it does look at the code preceding the code it's going to predict, and in my experience if you use double quotes there, it'll use them unless there's a reason to not.
